Cryptocurrency/Dash/dash-coincap-poloniex.5m.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
-# <bitbar.title>Dash tickers: Coinmarketcap and Poloniex</bitbar.title>
+# <bitbar.title>Dash tickers: Coincap.io and Poloniex.com</bitbar.title>
 # <bitbar.version>v1.0</bitbar.version>
```

Comment:
Please increment the version so people know this is a change.

Reply:
Done